themes/bootstrap3/templates/combined/jump-links.phtml

```diff
@@ -0,0 +1,10 @@
+<div class="combined-jump-links">
+  <span><?=$this->translate('combined_jump_links_intro')?></span>
+  <ul>
+    <?php foreach ($combinedResults as $section) : ?>
+      <li class="<?= $section['domId'] ?>">
```

Comment:
The 'domId' value needs to be run through `$this->escapeHtmlAttr` and the 'label' needs to be run through `$this->transEsc` to display things correctly.

Reply:
Thank you for the catch.  Fixed.